packages/constants/src/mcf.ts

```diff
@@ -438,3 +440,700 @@ export const SSEC_EQA_LIST: ISsecEqa[] = [
   {code: 'N9', description: 'Other Statement of Attainment, modular certificate or equivalent n.e.c.'},
   {code: 'XX', description: 'Not reported'},
 ];
+
+// Additional stop words are words that are common across all job description/titles
+export const CUSTOM_WORD_LIST = [
+  'pte',
+  'ltd',
+  'pte.',
+  'ltd.',
+  'private',
+  'limited',
+  'llc',
+  'llp',
+  'inc',
+  'inc.',
+  'co',
+  'co.',
+];
+
+// Stop words are retrieved from https://www.ranks.nl/stopwords
+export const STOP_WORDS = [
+  ...CUSTOM_WORD_LIST,
+  'a',
+  'about',
+  'above',
+  'after',
+  'again',
+  'against',
+  'all',
+  'am',
+  'an',
+  'and',
+  'any',
+  'are',
+  "aren't",
+  'as',
+  'at',
+  'be',
+  'because',
+  'been',
+  'before',
+  'being',
+  'below',
+  'between',
+  'both',
+  'but',
+  'by',
+  "can't",
+  'cannot',
+  'could',
+  "couldn't",
+  'did',
+  "didn't",
+  'do',
+  'does',
+  "doesn't",
+  'doing',
+  "don't",
+  'down',
+  'during',
+  'each',
+  'few',
+  'for',
+  'from',
+  'further',
+  'had',
+  "hadn't",
+  'has',
+  "hasn't",
+  'have',
+  "haven't",
+  'having',
+  'he',
+  "he'd",
+  "he'll",
+  "he's",
+  'her',
+  'here',
+  "here's",
+  'hers',
+  'herself',
+  'him',
+  'himself',
+  'his',
+  'how',
+  "how's",
+  'i',
+  "i'd",
+  "i'll",
+  "i'm",
+  "i've",
+  'if',
+  'in',
+  'into',
+  'is',
+  "isn't",
+  'it',
+  "it's",
+  'its',
+  'itself',
+  "let's",
+  'me',
+  'more',
+  'most',
+  "mustn't",
+  'my',
+  'myself',
+  'no',
+  'nor',
+  'not',
+  'of',
+  'off',
+  'on',
+  'once',
+  'only',
+  'or',
+  'other',
+  'ought',
+  'our',
+  'ours',
+  'ourselves',
+  'out',
+  'over',
+  'own',
+  'same',
+  "shan't",
+  'she',
+  "she'd",
+  "she'll",
+  "she's",
+  'should',
+  "shouldn't",
+  'so',
+  'some',
+  'such',
+  'than',
+  'that',
+  "that's",
+  'the',
+  'their',
+  'theirs',
+  'them',
+  'themselves',
+  'then',
+  'there',
+  "there's",
+  'these',
+  'they',
+  "they'd",
+  "they'll",
+  "they're",
+  "they've",
+  'this',
+  'those',
+  'through',
+  'to',
+  'too',
+  'under',
+  'until',
+  'up',
+  'very',
+  'was',
+  "wasn't",
+  'we',
+  "we'd",
+  "we'll",
+  "we're",
+  "we've",
+  'were',
+  "weren't",
+  'what',
+  "what's",
+  'when',
+  "when's",
+  'where',
+  "where's",
+  'which',
+  'while',
+  'who',
+  "who's",
+  'whom',
+  'why',
+  "why's",
+  'with',
+  "won't",
+  'would',
+  "wouldn't",
+  'you',
+  "you'd",
+  "you'll",
+  "you're",
+  "you've",
+  'your',
+  'yours',
+  'yourself',
+  'yourselves',
+];
+
+export const removeStopWords = (str = '') => {
+  const strArr = str.split(' ');
+  const sanitisedArr = remove(strArr, (word) => {
+    return !STOP_WORDS.includes(word);
+  });
+  return sanitisedArr.join(' ');
+};
+
+export const removeWordsInBracket = (str = '') => {
+  const re = /\(.*?\)/gi;
```

Comment:
Why do we need the `?` in this regex string?

Reply:
Given this implementation, we actually capture matches as follows:

```
((hello) hello)
// only ((hello) matches instead of ((hello) hello)
```

Is this what we expect?

Reply:
I think it's what we're expecting